### ISSUE TITLE
License / Attribution?

### ISSUE THREAD

**Issue by Santana18** [w3c/sdw#1036]
Hello,

my apologies, but could you please point me to the license / attribution rules for SOSA/SSN? I belief we can freely use, extend, modify it, but can't find the concrete license description. Could somebody provide me with a link to it?

Thanks!

**Comment by tidoust** [comment]
Hi @Santana18,

> my apologies, but could you please point me to the license / attribution rules for SOSA/SSN? I belief we can freely use, extend, modify it, but can't find the concrete license description. Could somebody provide me with a link to it?

SOSA/SSN is defined in the [Semantic Sensor Network Ontology](https://www.w3.org/TR/vocab-ssn/) W3C Recommendation, published under the [W3C Document License](https://www.w3.org/Consortium/Legal/2015/doc-license) (for what it's worth, the link to the license appears in the copyright statement at the end of the header). That document license explains attribution rules. It does not allow extending or modifying the vocabulary itself ("No right to create modifications or derivatives of W3C documents is granted pursuant to this license"). Extending here would typically mean creating new terms under the same namespace. Mixing the vocabulary with other vocabularies is certainly possible.

Vocabulary usage is encouraged. Participants of the Spatial Data on the Web Working Group, which developed the specification, have made certain licensing commitments to help create a Royalty-Free standard. See the [group's patent policy status](https://www.w3.org/2004/01/pp-impl/75471/status) for a list of organizations that made these commitments, and the [W3C Patent Policy](https://www.w3.org/Consortium/Patent-Policy-20170801/) for details about the licensing commitments.

**Comment by Santana18** [comment]
It was pointed out to me that the ontologies themselves refer these two:

dcterms:license <http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document> ;
dcterms:license <http://www.opengeospatial.org/ogc/Software> 

So the ontology files themselves are allowed to be modified as desired.  But you suggest that I am not supposed to - for (arbitrary not recommended) example - rename sosa:Observation into sosa:Measurement and publish that as a text and say this is a new version of Sosa. But it would definitely ok to for example - (again arbtitrary) -  add a "vehicle" and this vehicle I can connect with or use with all parts of ssn/sosa, as I like, and in the ontologies for simplicity I can also connect it how I like. E.g. I can use all the vocabulary as desired, but should just clarify that I use and extend the ontologies, and not claim that I make a new version of the specification, which is completely understandable.

**Comment by tidoust** [comment]
Right, sorry if I made things unnecessarily confusing. W3C considers that the normative definition of the ontologies is the spec itself and that the ontologies published under the namespace URI are "just" code examples (what the W3C Document License refers to as "Code Components"), and these are covered by the W3C Software and Document License, which allows derivative work.

For what it's worth, most groups at W3C now also publish specs themselves under the W3C Software and Document License. This group did not, mostly because it started its work before that really became the norm.

**Comment by Santana18** [comment]
Just to be sure, so you seem to agree with my interpretation of how I am allowed to reuse it? :-)

**Comment by tidoust** [comment]
Yes, your interpretation seems good to me!
